app/api/exhibitors.py

```diff
@@ -30,13 +30,24 @@ def query(self, view_kwargs):
         query_ = event_query(query_, view_kwargs)
         return query_
 
+    def after_create_object(self, exhibitor, data, view_kwargs):
+        if data.get('banner_url'):
+            start_image_resizing_tasks(exhibitor, data['banner_url'])
+
     view_kwargs = True
-    methods = ['GET']
+    methods = ['GET', 'POST']
```

Comment:
Wrong, this is API is just for querying exhibitors. For creating them, ExhibitorListPost is present